_data/cards.yml

```diff
@@ -0,0 +1,37 @@
+develop:
+  - link: "/cockroachcloud/quickstart.html"
+    title: "Create a new cluster"
+    text: "Create and connect to a new CockroachDB cluster."
+  - link: "/stable/install-client-drivers.html"
+    title: "Install drivers"
+    text: "Install client drivers for your application."
+  - link: "/stable/connect-to-the-database.html"
+    title: "Connect to a cluster"
+    text: "Get details on how to connect to a CockroachDB cluster using your favorite language and framework."
+  - link: "/stable/example-apps.html"
```

Comment:
when I used the dropdown to select the v23.1XXX of this page, these links still bring me to v22.2.* versions of these pages

looking at this I can see why of course!

is there a way to have these links point at docs pages whose version matches?

Reply:
I think I figured out a way to do this.